### PR TITLE
[WIP] Patch5/fix travis build failing

### LINKED ISSUE #149: Fix Travis tests
**I'm submitting a ...**
- [X] bug report

**Current behavior:**

Currently, travis is failing.
https://travis-ci.org/fossasia/badgeyay/builds/278248341?utm_source=github_status&utm_medium=notification

**Expected behavior:**

Travis should run all the tests #147 and succeed.

**Steps to reproduce:**

Please create a pull request to work on this issue and submit commits which fix the errors.

### PR DESCRIPTION
**Pull request by vaibhavsingh97**
Fixes #149 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] I have added necessary documentation (if appropriate)

#### Root Cause of Travis Failure
When we run tests on our desktop than we first run `python app/main.py` and then run `nosetests app/tests/test.py` and test run successfully but in travis this is not the case and travis try to run only `nosetests app/test/test.py` and when in test try to request url and it's not found than all the tests fail

#### Fix for this error
If we can run `python app/main.py` and `nosetests app/tests/test.py` parellel than we can fix this issue.

@fossasia/badgeyay-admin What are your opinions?

### CODE REVIEW

**Comment by vaibhavsingh97** [comment]
Please don't merge as it's not working as expected 😅

File: .travis.yml [hunk 1]
```diff
@@ -1,12 +1,35 @@
...
+before_script:
+- chmod +x ./start_server.sh
 
 script:
-- nosetests app/tests/test.py -v --with-coverage
+- ./start_server.sh
+- python app/main.py &
+- sleep 3
+- nosetests app/tests/test.py -v --with-coverage &
+- sleep 3
```
Comment (S2606) [inline .travis.yml]: @vaibhavsingh97 if you have stated both `python app/main.py &` and ` nosetests app/tests/test.py -v --with-coverage &` here,so whats the use of start_server.sh script then?

Comment (vaibhavsingh97) [reply .travis.yml]: @S2606 I am testing so basically what i was trying to start `python app/main.py` and let that be in background and start testing than i would probably kill all after successfull completion of test but when i used `start_server.sh ` travis not recognizing any file i installed from requirements.txt so what i did is to test that what is happening that i let the start_server.sh remain and added the same thing in travis 😅

**Comment by vaibhavsingh97** [comment]
@fossasia/badgeyay-admin Anyone can help me or collaboate to fix #149 as it's a priority issue.

**Comment by S2606** [comment]
@vaibhavsingh97 I am working on the issue

**Comment by vaibhavsingh97** [comment]
If you find the solution please update this PR 😄

**Comment by vaibhavsingh97** [comment]
Closing this PR as #262 will fix the issue 😅

**Pull request closed without merging**